packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/codeActions/impl/MongoDbRunQueryActionUiTest.kt

```diff
@@ -20,12 +21,15 @@ class MongoDbRunQueryActionUiTest {
         remoteRobot: RemoteRobot,
         url: MongoDbServerUrl,
     ) {
+        // This is our safety net to ensure we start with a clean slate
+        remoteRobot.openDatabaseToolWindow().removeAllDataSources()
+        remoteRobot.closeRightToolWindow()
```

Comment:
In 99.99% of cases, this will be a no-op but I am keeping this as a safety net to reduce the flakes.